modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/singlecelltrace/v0.1.2.json

```diff
@@ -0,0 +1,21 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/simulation/singlecelltrace/v0.1.0/shapes/"
```

Comment:
WHat is the purpose of this schema ?

Reply:
To hold the trace file. File name at the moment will contain the params used to produce the trace. Later we'll extract that info to a proper properties of simulation activity.

Reply:
I see. Can we name it singlecellsimulationtrace ? There is a trace schema in the ephys namespace (we will need to rename it as well) which is a single cell one as well .

Reply:
I depends if the properties in the ephys trace (from recordings) - like channels- schema are relevant in the context of the simulation.

Reply:
Hi @apdavison ! In principle, that would be a good solution. However, we might need a generic trace schema which would be imported by a schema for a biological and a simulated trace respectively should they require different property shapes. Currently, the ephys trace schema contains a property shape for **channel**, which is required. I don't know whether a simulated trace has an equivalent to the recording channel a biological trace has, @genric ?

Reply:
Yep, I found that ephys trace was way too constrained for simulation use-case, so I decided to go for a new entity instead.

Reply:
fair enough! (I like the idea of a generic trace schema, but perhaps for a later PR...)